src/lib/Libattr/master_sched_attr_def.xml

```diff
@@ -250,7 +250,167 @@
+        <member_at_action>set_sched_iteration</member_at_action>
+        <member_at_flags><both>MGR_ONLY_SET</both></member_at_flags>
+        <member_at_type><both>ATR_TYPE_LONG</both></member_at_type>
+        <member_at_parent>PARENT_TYPE_SCHED</member_at_parent>
+        <member_verify_function>
+        <ECL>verify_datatype_long</ECL>
+        <ECL>NULL_VERIFY_VALUE_FUNC</ECL>
+        </member_verify_function>
+   </attributes>
+   <attributes>	/* sched_user */	
+	<member_name><both>ATTR_sched_user</both></member_name>		<!-- "partition" -->
+	<member_at_decode>decode_str</member_at_decode>
+	<member_at_encode>encode_str</member_at_encode>
+	<member_at_set>set_str</member_at_set>
+	<member_at_comp>comp_str</member_at_comp>
+	<member_at_free>free_str</member_at_free>
+	<member_at_action>set_sched_user</member_at_action>
+	<member_at_flags><both>MGR_ONLY_SET</both></member_at_flags>
+	<member_at_type><both>ATR_TYPE_STR</both></member_at_type>
+	<member_at_parent>PARENT_TYPE_SCHED</member_at_parent>
+	<member_verify_function>
+	<ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
+	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
+	</member_verify_function>
+   </attributes>
+   <attributes> /* sched_comment */
+        <member_name><both>ATTR_sched_comment</both></member_name>                   <!-- "comment" --> 
+        <member_at_decode>decode_str</member_at_decode>
+        <member_at_encode>encode_str</member_at_encode>
+        <member_at_set>set_str</member_at_set>
+        <member_at_comp>comp_str</member_at_comp>
+        <member_at_free>free_str</member_at_free>
+        <member_at_action>NULL_FUNC</member_at_action>
+        <member_at_flags><both>MGR_ONLY_SET</both></member_at_flags>
+        <member_at_type><both>ATR_TYPE_STR</both></member_at_type>
+        <member_at_parent>PARENT_TYPE_SCHED</member_at_parent>
+        <member_verify_function>
+        <ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
+        <ECL>NULL_VERIFY_VALUE_FUNC</ECL>
+        </member_verify_function>
+    <attributes>  /* job_sort_formula */
+        <member_name><both>ATTR_job_sort_formula</both></member_name>
+        <member_at_decode>decode_formula</member_at_decode>
+        <member_at_encode>encode_str</member_at_encode>
+        <member_at_set>set_str</member_at_set>
+        <member_at_comp>comp_str</member_at_comp>
+        <member_at_free>free_str</member_at_free>
+        <member_at_action>validate_job_formula</member_at_action>
+        <member_at_flags><both>READ_WRITE</both></member_at_flags>
+        <member_at_type><both>ATR_TYPE_STR</both></member_at_type>
+        <member_at_parent>PARENT_TYPE_SCHED</member_at_parent>
+        <member_verify_function>
+        <ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
+        <ECL>NULL_VERIFY_VALUE_FUNC</ECL>
+        </member_verify_function>
+   </attributes>
+   <attributes> /* BackfillDepth */
+        <member_name><both>ATTR_backfill_depth</both></member_name>             <!-- "backfill_depth" -->
```

Comment:
As suggested since we are moving jobs_sort_formula ,backfill_depth to policy object in future, we are leaving them on the server object itself.